CMakeLists.txt

```diff
@@ -1,32 +1,80 @@
-cmake_minimum_required(VERSION 3.1)
-
-
-set(CMAKE_DISABLE_SOURCE_CHANGES ON)
-set(CMAKE_DISABLE_IN_SOURCE_BUILD ON)
+cmake_minimum_required(VERSION 3.9)
 
 project(NetKet)
 
-set (CMAKE_CXX_STANDARD 11)
-set (NETKET_SOURCE_DIR ${CMAKE_CURRENT_SOURCE_DIR}/NetKet/)
-set (EXTERNAL_INCLUDE_DIR ${CMAKE_CURRENT_SOURCE_DIR}/External/)
+include(CTest) # adds option BUILD_TESTING (default ON)
+include(FetchContent)
+
+# set (CMAKE_CXX_STANDARD 11)
```

Comment:
Why this is not necessary anymore?

Reply:
Indeed it seems that compilation fails without this?

Reply:
You're very right. I meant to use the `target_compile_feature` instead, but never actually got to adding it... Should be fixed in the last commit. The reason why this approach is preferable nowadays is that it doesn't use global variables.